From: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>
Date: Thu, 26 Mar 2020 21:16:18 +0000
Subject: src/xz/xz.1: Correct misused two-fonts macros

Output is from: test-groff -b -e -mandoc -T utf8 -rF0 -t -w w -z

  [ "test-groff" is a developmental version of "groff" ]

Input file is ./src/xz/xz.1

<src/xz/xz.1>:408 (macro BR): only 1 argument, but more are expected
<src/xz/xz.1>:1009 (macro BR): only 1 argument, but more are expected
<src/xz/xz.1>:1743 (macro BR): only 1 argument, but more are expected
<src/xz/xz.1>:1920 (macro BR): only 1 argument, but more are expected
<src/xz/xz.1>:2213 (macro BR): only 1 argument, but more are expected

  Output from nroff and troff is unchanged, except for a font change of a
full stop (.).

Signed-off-by: Bjarni Ingi Gislason <bjarniig@rhi.hi.is>
---
 src/xz/xz.1 | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/xz/xz.1 b/src/xz/xz.1
index 540d136..4a11ce3 100644
--- a/src/xz/xz.1
+++ b/src/xz/xz.1
@@ -405,7 +405,7 @@ to the target file.
 .IP \(bu 3
 When used with
 .B \-\-decompress
-.BR \-\-stdout
+.B \-\-stdout
 and
 .B xz
 cannot recognize the type of the source file,
@@ -1006,7 +1006,7 @@ until the details have been decided.
 .RE
 .IP ""
 For 32-bit
-.BR xz
+.B xz
 there is a special case: if the
 .I limit
 would be over
@@ -1740,7 +1740,7 @@ See the section
 .B "ROBOT MODE"
 for details.
 .TP
-.BR \-\-info\-memory
+.B \-\-info\-memory
 Display, in human-readable format, how much physical memory (RAM)
 .B xz
 thinks the system has and the memory usage limits for compression
@@ -1917,7 +1917,7 @@ Compressed size of the file
 Uncompressed size of the file
 .IP 6. 4
 Compression ratio, for example
-.BR 0.123.
+.BR 0.123 .
 If ratio is over 9.999, three dashes
 .RB ( \-\-\- )
 are displayed instead of the ratio.
@@ -2210,7 +2210,7 @@ is practically a superset of
 .BR lzma ,
 .BR unlzma ,
 and
-.BR lzcat
+.B lzcat
 as found from LZMA Utils 4.32.x.
 In most cases, it is possible to replace
 LZMA Utils with XZ Utils without breaking existing scripts.
